editoast/src/core/conflict_detection.rs

```diff
@@ -13,18 +14,17 @@ use super::stdcm::WorkSchedule;
 
 editoast_common::schemas! {
     ConflictDetectionResponse,
-    Conflict,
+    ConflictResponse,
```

Comment:
Out of curiosity, what motivated this rename? (It would've been nice to have a separate commit just for the rename, with an explanation in the commit message.)